web-portal/backend/src/utils/utils.service.ts

```diff
@@ -151,8 +151,7 @@ export class UtilsService {
         });
 
         const parsedLogs: IParsedLog[] = logs.map((log: any) => ({
-            tenantId: web3.utils
-                .toAscii(log?.args?._identifier!)
+            tenantId: fromHex(log?.args?._identifier!, 'string')
                 .replaceAll(`\x00`, ''),
```

Comment:
One thing I thought of here is what happens if there are zeros in the identifier that get removed?

Reply:
doesn't work that way for un-padded zeroes! (hopefully)